thriftbp/faults.go

```diff
@@ -31,11 +31,13 @@ func NewClientFaultMiddleware(clientName, address string) clientFaultMiddleware
 
 type thriftHeaders struct{}
 
+var _ faults.Headers = &thriftHeaders{}
+
 // Lookup returns the value of the header, if found.
-func (h thriftHeaders) Lookup(ctx context.Context, key string) (string, error) {
+func (h *thriftHeaders) LookupValues(ctx context.Context, key string) ([]string, error) {
 	header, ok := thrift.GetHeader(ctx, key)
 	if !ok {
-		return "", nil
+		return []string{}, nil
```

Comment:
nit:
```suggestion
		return nil, nil
```
see https://google.github.io/styleguide/go/decisions#nil-slices

Reply:
Thanks for the pointer! Done.